colrev/env/local_index.py

```diff
@@ -129,7 +130,82 @@ def __get_sqlite_cursor(self, *, init: bool = False) -> sqlite3.Cursor:
         self.sqlite_connection = sqlite3.connect(self.SQLITE_PATH, timeout=90)
         self.sqlite_connection.row_factory = self.__dict_factory
         return self.sqlite_connection.cursor()
-        # raise colrev_exceptions.ServiceNotAvailableException(dep="local_index")
+
+    def load_journal_rankings(self) -> None:
+        """Loadsd journal rankings into sqlite database"""
+        rankings_csv_path = (
+            str(Path(__file__).parents[1])
+            / Path("template")
+            / Path("ops")
+            / Path("journal_rankings.csv")
+        )
+        conn = sqlite3.connect(self.SQLITE_PATH)
+        data_frame = pd.read_csv(rankings_csv_path, encoding="cp850")
+        data_frame.to_sql("rankings", conn, if_exists="replace", index=False)
+        conn.commit()
+        conn.close()
+
+    def search_in_database(self, journal: typing.Optional[typing.Any]) -> str:
+        """Searches for journalranking in database"""
+        cur = self.__get_sqlite_cursor(init=False)
```

Comment:
At this point, a more elegant solution would be to run a single query (`SELECT * FROM rankings WHERE journal_name=?`).